src/utils/__test__/RunnersUtils.spec.js

```diff
@@ -61,3 +61,86 @@ describe('patchScenarioParameterValues', () => {
     spyConsoleWarn.mockClear();
   });
 });
+
+describe('update parentId on delete', () => {
```

Comment:
Can we add a test to check the behavior when a scenario defined as `parentId` is not defined in the runners list? (this may happen when users have access to a child scenario but not its parent, because of ACL restrictions)

Example, with this structure of runners, but where `r-root` would not be shared with the current user:
```
 - `r-root`
   - `r-toremove`
     - `r-childscenario`
```